benchmark/index.js

```diff
@@ -119,6 +160,7 @@ test('migrate (+db1)', async (t) => {
       fs.appendFileSync(reportPath, `| Migrate (+db1) | ${duration}ms |\n`)
       await sleep(2000) // wait for new log FS writes to finalize
       sbot.close(() => {
+        t.end()
```

Comment:
I noticed quite a few of these had this missing. Seemed like a good idea to have, or is there an implicit t.end?

Reply:
Oh, sorry that this is a bit mysterious, but we're using promises in these `async-await` tests, which means we don't need to do a t.end(), see https://github.com/substack/tape#testname-opts-cb (search "promise")

Reply:
Ok, will remove those then